app/controllers/webauthn_credentials_controller.rb

```diff
@@ -26,6 +36,7 @@ def callback
 
   def destroy
     webauthn_credential = current_user.webauthn_credentials.find(params[:id])
+
     if webauthn_credential.destroy
       flash[:notice] = t(".webauthn_credential.confirm_delete")
```

Comment:
Missing deletion for recovery codes.

Reply:
Also for totp it looks like. Adding these in now.

Reply:
Oh right - can't believe I forgot about that. Thank you!